modules/nodes-nodes-resources-configuring-auto.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-resources-configuring.adoc
+
+[id="nodes-nodes-resources-configuring-auto_{context}"]
+= Automatically allocating resources for nodes
+
+{product-title} can automatically determine the optimal `system-reserved` CPU and memory resources for nodes associated with a specific machine config pool and update the nodes with those values when the nodes start.  
+
+To automatically determine and allocate the `system-reserved` resources on nodes, create a `KubeletConfig` custom resource (CR) to set the `autoSizingReserved: true` parameter. A script on each node calculates the optimal values for the respective reserved resources based on the installed CPU and memory capacity on each node. The script takes into account that increased capacity requires a corresponding increase in the reserved resources. 
+
+Automatically determining the optimal `system-reserved` settings ensures that your cluster is running efficiently and prevents node failure due to resource starvation of system components, such as CRI-O and kubelet, without your needing to manually calculate and update the values. 
+
+This feature is disabled by default.
+
+.Prerequisites 
+
+. Obtain the label associated with the static `MachineConfigPool` object for the type of node you want to configure.
+Perform one of the following steps:
+
+.. View the machine config pool:
++
+[source,terminal]
+----
+$ oc describe machineconfigpool <name>
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc describe machineconfigpool worker
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
```

Comment:
Does this have a `name`? I think all objects always include at least a `name` field.